frontend/src/components/UserTables/DispatcherTable.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState, useEffect, Dispatch, SetStateAction } from 'react';
+import DispatcherForm from '../UserForms/DispatcherForm';
+import { Dispatcher } from '../../types/index';
+import styles from './table.module.css';
+
+const DispatcherManager = () => {
+
+  const [dispatchers, setDispatchers] = useState<Dispatcher[]>([]);
+
+  const getCurrentDispatchers = async () => {
+    const dispatchersData = await fetch('/dispatchers')
+      .then((res) => res.json())
+      .then((data) => data.data);
+    setDispatchers(
+      dispatchersData.map((dispatcher: Dispatcher) => ({
+        id: dispatcher.id,
+        firstName: dispatcher.firstName,
+        phoneNumber: dispatcher.phoneNumber,
+        email: dispatcher.email,
+        accessLevel: dispatcher.accessLevel
+      })),
```

Comment:
Why map to a new object if it will contain all of the same attributes as the entire Dispatcher object? Don't need this at all. You could just do `setDispatchers(dispatcherData)`